script/db_workbench.py

```diff
@@ -6,11 +6,11 @@
 from flask_bcrypt import Bcrypt
 from flask_jwt_extended import JWTManager, create_access_token
 
+sys.path.insert(0, "/appdata")
```

Comment:
this is required to import create_app and connect 
Idk if there's a better way to do this though

Reply:
Whoops, my bad this was probably caused by Isort.